### PR TITLE
feat(polyfills): add polyfill for composedPath() to work with IE

### PR DESCRIPTION
**Pull request by ps011**
This PR will add a polyfill for Event.composedPath() function to get that working in Internet Explorer. 
Since this is [not supported by IE](https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath) and we need support for that

### CODE REVIEW

File: src/client/polyfills/composedpath.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    this.path = [];
+    while (target.parentNode !== null) {
+      this.path.push(target);
+      target = target.parentNode;
+    }
+    this.path.push(d, w);
+    return this.path;
+    }
+  }
+})(Event.prototype, document, window);
```
Comment (arjunyel) [inline src/client/polyfills/composedpath.js]:
```suggestion
})(Event.prototype, document, window);

```

Add newline at end

**Comment by ps011** [comment]
@adamdbradley , anything else need to be done in this ?

**Comment by simonhaenisch** [comment]
Was about to start working on this, however I just saw that this polyfill was already added in the meantime:

https://github.com/ionic-team/stencil/blob/62839d8e81c1d68bbc177ff35c4f1dd03cad8496/src/client/polyfills/dom.js#L55-L73

Thanks for the PR though!

**Comment by ps011** [comment]
@simonhaenisch  was the polyfill added after 0.15.2 ?

**Comment by simonhaenisch** [comment]
It was added in `v1.0.0-beta.8` (381ded9d08fec13c8c39fcc7015e358a5ee48e6d).

**Pull request closed without merging**